.github/workflows/verify-gradle-wrapper.yaml

```diff
@@ -0,0 +1,11 @@
+name: Validate Gradle wrapper
+on: [pull_request, push]
+jobs:
+  build_pr:
+    if: github.repository_owner == 'jenkinsci'
```

Comment:
Does this work on forks with this if?

Reply:
Yes: https://github.com/jenkinsci/acceptance-test-harness/actions/workflows/verify-gradle-wrapper.yaml

The if-clause context refers to the owner of the repository where the workflow is running, not where the PR is coming from.
The job would only be skipped if you file a PR _on_ your fork _towards_ your fork.